### ISSUE TITLE
Disable annoying right click + mouse wheel document switching

### ISSUE THREAD

**Issue by MediaEU** [notepad-plus-plus/notepad-plus-plus#9735]
Notepad++ v7.9.1
If you have multiple file open and you coincidentally press right mouse click + scroll it opens a super annoying "tooltip/window" where you can switch between opened files.

The main issue is you can't disable this behavior I would say it's a bug not just a behavior.

Steps to Reproduce the Issue
1.  Open multiple files
2.  Try click right mouse + mouse scroll
3. Voila there you have it a super super annoying window pops up

Expected Behavior
I would expect that nothing happend

Actual Behavior
Super super super annoying behavior happens

![dumbo_popup](https://user-images.githubusercontent.com/44848420/113741637-24ea5d80-9702-11eb-8832-9fe5ef07ae62.png)

**Comment by sasumner** [comment]
Why don't you just not press the right mouse button and then scroll?

This was decided not to be dealt with with the closure of #2603.
In the unlikely event that this is to be revisited, 2603 can be reopened.

**Comment by MediaEU** [comment]
"Why don't you just not press the right mouse button and then scroll?"
sasumner is that post of your life?
I was serious about that and didn't act like a little child as you do

**Comment by sasumner** [comment]
Let me rephrase, because I'm trying to understand:  Why would you do this action if it annoys you?
If you want, say, the context menu, then right-click (but don't scroll the wheel).
If you want to, say, scroll the document contents, then scroll the wheel (but don't right-click).
What is the justifiable use case for doing both of these actions simultaneously (so that there is the possibility of it annoying you)?

**Comment by Ekopalypse** [comment]
@MediaEU 
Strange, my 7.9.1 doesn't have this popup if I disable the document switcher.

**Comment by MediaEU** [comment]
@Ekopalypse 
 That did the job done. thx   

uncheck     
Settings->Preferences->MISC->Document Switcher: ☑Enable

**Comment by mcreereg** [comment]
From what I can tell, there's still no option to disable right-click + scroll behavior that changes your active tab.  You can disable the popup from displaying, but you can't stop the editor from changing which tab is active.

Any chance that either #2603 or #9735 can be re-opened as a feature request to have an option to disable this behavior completely?

> Why don't you just not press the right mouse button and then scroll?

I never _mean_ to do this.  I do this accidentally a couple times a week when I try to scroll several pages rapidly with my big clumsy hands.  I understand that there are other ways to scroll through a document, but sometimes spamming the scroll wheel is how I want to do it, and this tab-switching behavior ends up biting me.
